doc/ext/math.rst

```diff
@@ -44,6 +44,14 @@ or use Python raw strings (``r"raw"``).
 
    Example: ``'Eq.{number}'`` is rendered as ``Eq.10``
 
+.. confval:: math_numfig
+
+   If ``True``, displayed math equations are numbered across pages using numfig.
+   The :confval:`numfig` config value must be enabled and
+   :confval:`numfig_secnum_depth` is respected.  The ``:eq:`` role must be used
+   to refererence these equation numbers, not the ``:numref:`` role.
+   Default is ``False``.
```

Comment:
I think this is very useful, so I propose you to enable this by default. What do you think?

BTW, I think now this option is available only for HTML, not LaTeX output. Hence it should be mentioned.

Reply:
Perhaps make clearer option applies only to html like target (html, epub, singlehtml, ...). And ``math_numfig_for_eq`` might be clearer as name.

Reply:
I have added the limitation to html-like output.
I'd be happy to enable it by default.
Happy also with the name change if others agree.

Reply:
Can you use ``:rst:role:`eq``` and ``:rst:role:`numref``` mark-up?

About name change I have no strong opinion.

Reply:
@jfbu this option also effect to block level equations (a.k.a displaymath node). So I feel `for_eq` is not good. Also, `math_` prefix describes clarify its purpose to me.

Reply:
LGTM for `math_numfig` option name!

Reply:
Took care of :rst:role: markup.  Also changed default to True as requested.  I felt a slight rewording was appropriate in this case, which I applied.